packages/storybook8/stories/INTERNAL/VideoEffects/index.stories.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Meta } from '@storybook/react';
+import { SelectableVideoEffects } from './snippets/SelectableVideoEffects.snippet';
+import { VideoBackgroundEffectsPicker } from './snippets/VideoBackgroundEffectsPicker.snippet';
+
+export { VideoEffects } from './VideoEffects.story';
+
+export const VideoBackgroundEffectsPickerExampleDocsOnly = {
+  render: VideoBackgroundEffectsPicker
+};
+
+export const SelectableVideoEffectsExampleDocsOnly = {
+  render: SelectableVideoEffects
+};
+
+const meta: Meta = {
+  title: 'Components/Internal/Video Effects',
```

Comment:
nitpick: Just a thought, since main strips everything out of `internal` folder, we probably don't need Internal path in the title, and just list it as if it was a normal component

Reply:
In my opinion, I think it is good to have internal folder so that we know which components are not available in the library. We also follow the same thing in V6. Happy to chat further and know your thoughts on this!
<img width="323" alt="Screenshot 2024-08-08 at 1 23 14 PM" src="https://github.com/user-attachments/assets/02917d47-8e94-4ef4-b223-f5f630225c73">